### DISCUSSION TITLE
Registered tournaments notification reminder

### DISCUSSION THREAD

**Discussion by shuvashish76** [gbtami/pychess-variants#858] [Ideas]
https://www.pychess.org/calendar

Tournament calendar is a really nice feature as users don't need to come pychess frequently just to check their upcoming favourite/preferred tournaments.

- It would be nice to get such registered tournament notification directly on lichess as inbox message.
  - Configuration : option to turn registered tournament notification reminder on/off
    - option to set custom time for notification "before 30mins", "before 1hr", "before 3hr", "before 6hr". I think such 4 options should be enough as we expect the user to be active on lichess.

---

Here is an example of automated notification from [lichess4545](https://lichess.org/@/lichess4545) account👇🏿
![Screenshot_2022-02-19-23-27-28](https://user-images.githubusercontent.com/48226312/154813041-3e604ab8-eb96-4ef8-a8a0-bc4d0dacff34.jpeg)

_(they use https://github.com/Lichess4545/Chesster bot to integrate lichess with their slack team)_

I think for our usecase 1 notification for each tournament is enough.

Limitations:
 - Disable notification if the user joined any tournament start in 30mins, as we expect him to play the arena. And getting such notification unnecessarily could be annoying to some players.

(_#811 is a similar feature request but the purpose & usecase is complete different than this one_ Team is for notifying all tournaments to pychess team players on lichess. But this one is for "user-preferred registered tournaments custom-reminder".)

Let me know if you disagree.

ThankYou🙏🏿

**Comment by gbtami** [comment]
I'm not sure this kind of direct messaging not will be considered API misuse or spamming on lichess.
We already have something similar in our Discord. Users can set preferred variants in #self-roles channel to get notifications.

**Comment by shuvashish76** [comment]
Umm... Then I'd suggest to disable that notification option on pychess by_default for all & new users.
I don't think most privacy oriented FOSS users would like to use proprietary Discord service for getting notifications. Though I understand we've lots of users on Discord & makes no sense to maintain multiple platforms for that.

If you feel it's worth for discussion (for opinions of other users) then please convert this issue to a [discussion](https://github.com/gbtami/pychess-variants/discussions) else close this issue from your side :)

**Comment by gbtami** [comment]
We send daily notifications to members of https://lichess.org/team/pychess-tournaments 
I think it is enough for now.
